### DISCUSSION TITLE
Category Blog displaying links to subcategories Astroid Template Zero

### DISCUSSION THREAD

**Discussion by puffermedia** [templaza/astroid-framework#789] [Q&A] [answered]
I have determined that this is a change to Astroid Template Zero 3.1.0. 
There is a blog link in my main menu for Category Blog | blog.
It used to be that on the top level it would just show the most recent blog post. But with the most recent update the Category Blog view now shows the h3 "Sub Categories" with links to whatever the sub categories are. It is not a Joomla change, because I updated to the most recent Joomla on a dev site *without* updating Astroid and it didn't change this. 
![blog-view-screenshot-01](https://github.com/user-attachments/assets/c496ea21-023b-4153-936a-b464363a1c53)
Once I updated the Astroid Template Zero this started showing on my Dev site as well
![blog-view-screenshot-02](https://github.com/user-attachments/assets/52a4a922-5a62-4adc-8e03-755753dce5a3)

I don't see where in the Admin to change this behavior. 

 I donated to the Ko-fi just now because I'm requesting support. :)

**Comment by puffermedia** [comment]
For the time being I've just set 
.com-content-category-blog__children.cat-children 
to display:none

**Comment by sonvnn** [comment, marked as answer]
@puffermedia It's an Joomla's feature which was forgot by Joomdev when they develop Zero. 
To turn it off go to Menu -> Menu Item -> Category -> Subcategory Levels -> Choose "none"
![Screen Shot 2024-09-04 at 15 10 19](https://github.com/user-attachments/assets/e3631d5a-8e3d-45f6-9ec7-bbddbdeef0ae)

**Reply by puffermedia** [reply]
Ah, thanks. It's bit of a pain that I have to manually do that with every category that has a sub-category, but in the case of my blog I guess that is only every year. 

Good to know for future development.
